calculation/CopyValueIntersectingFeature.md

```diff
@@ -0,0 +1,48 @@
+# Copy a Value from an Intersecting Feature
+
+This calculation attribute rule copies a value from an intersected feature to the edited feature.
+
+## Use cases
+
+To copy a life cycle, diameter, district, etc from a feature that created or edited feature intersects.
+
+## Workflow
+
+Using ArcGIS Pro, use the Add Attribute Rule geoprocessing tool to define this rule on a feature class and optionally on a subtype in that feature class.  Use the following values when defining the rule, the other options are not required or depend on your situation.
+  
+  - **Rule Type:** Calculation
+  - **Triggering Events:** Insert, Update
+
+
+
+## Expression Template
+
+This Arcade expression will return the a value from an intersected feature
+
+```js
+// This rule will populate the edited features field with a value from an intersecting feature
+
+// Value to copy from the intersected feature
+var intersecting_field = "maintby";
+//Field rule is assigned to in the Attribute Rule
+var feature_field = "maintby";
+
+// Create feature set to the intersecting class using the GDB Name
+var intersecting_featset = FeatureSetByName($datastore, 'WaterLine', [intersecting_field], true);
```

Comment:
I was under the impression that passing vars for field names as opposed to a string literal causes issues in Pro. This  expression is good as far as I'm concerned, but I just wanted to make sure this works as expected.

Reply:
Variables for FeatureClass does not work, variables for fields does.  Let me build a sample FGDB like I did for the other rule so we can verify

Reply:
Want me to move to a Attribute Rule Calculation folder?

Reply:
Verified and added sample, added a few more from our sewer config too

Reply:
> Want me to move to a Attribute Rule Calculation folder?

That would be great!

Reply:
Moved to own folder and updated readme